cmd/activator/request_log.go

```diff
@@ -0,0 +1,45 @@
+package main
```

Comment:
needs license header

Reply:
Also, why is this next to the binary instead of in a library?

Reply:
They are only used to initialize the binary. Think of them being in main.go but didn't want to increase main.go's size too much.